guides/quick-start.md

```diff
@@ -0,0 +1,61 @@
+## At a glance
+
+There are two main routes to getting started with Mailchimp Open Commerce: installing the platform on your local computer or on a server. This guide will focus on the former, which will allow you to explore the main features of Open Commerce.
+
+In this guide, we’ll set up a full local instance of the Open Commerce platform, including core plugins provided by Mailchimp. We’ll walk through cloning and starting the Open Commerce development platform, registering an account, and creating your first shop. 
+
+## What you’ll need
+
+  * [Docker](https://www.docker.com/)
+
+  * [Docker Compose](https://docs.docker.com/compose/)
+
+  * Windows users: [WSL 2](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and [Docker for WSL](https://docs.docker.com/docker-for-windows/wsl/)
+
```

Comment:
@alexmarple not sure if Git is installed by default in all platforms, sometimes is not. maybe is a good thing to mention it. (even if we are on github)
[Git Guide](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)

In Mac OS, Xcode command line tools will install Git and Make for example

Another thing worth mentioning is to set the memory in docker to at least 4GB, otherwise reaction-admin will fail on build.

Reply:
@delagroove Since we are just moving these repos into `/guide`, I would be fine in merging these changes as is. We can open a separate PR for these modification. 
But we can also make these changes here. Either way, it seems helpful to have these things included (especially docker memory stuff)